src/style.css

```diff
@@ -1,3 +1,161 @@
+* {
+  box-sizing: border-box;
+}
+
+h2 {
+  margin: 0;
+  padding: 0;
+  border: 0;
+  outline: 0;
+  vertical-align: baseline;
+  background: transparent;
+  font-size: 100%;
+}
+
+p {
+  margin: 0;
+  padding: 0;
+  border: 0;
+  outline: 0;
```

Comment:
```
padding: 0;
border: 0;
outline: 0;
background: transparent;
```
There is no default styling in the browser for these properties, so you can delete it as redundant